dyno-core/src/main/java/com/netflix/dyno/connectionpool/ConnectionPool.java

```diff
@@ -148,4 +148,6 @@
      */
     HealthTracker<CL> getHealthTracker();
 
+    boolean isIdle();
```

Comment:
isIdle applicable to both the shadow and the main pool for the primary cluster? IF it is only for the shadow, can we rename it to isIdleShadow()?